cmd/extractDiffs.go

```diff
@@ -29,34 +34,38 @@ func init() {
 	rootCmd.AddCommand(extractDiffsCmd)
 }
 
+func getContractAddresses() []string {
+	LogWithCommand.Info("Getting contract addresses from config file")
+	contracts := viper.GetStringMap("contract")
+	var addresses []string
+	for contractName := range contracts {
+		address := viper.GetStringMapString("contract." + contractName)["address"]
+		addresses = append(addresses, address)
```

Comment:
Wonder if it's worth tossing a `panic` in here if `address == ""`

Reply:
Good call  - just added the CreateFilterQuery method that we have in prod as part of fe275dc